tsconfig.json

```diff
@@ -0,0 +1,100 @@
+{
```

Comment:
Generated with `tsc --init`. Anything commented out is a default that TS applies.